base/setup/usetup/usetup.c

```diff
@@ -58,6 +58,7 @@ WCHAR DefaultLanguage[20];
 WCHAR DefaultKBLayout[20];
 BOOLEAN RepairUpdateFlag = FALSE;
 HANDLE hPnpThread = INVALID_HANDLE_VALUE;
+LONG ProgressBarWidth = 0;
```

Comment:
Wrong: this is a global variable. That data should be easily computable from the different "Bar->xxx" fields within the callback.